app/views/home/about.html.erb

```diff
@@ -87,7 +87,7 @@
         <p>Any code snippets included in <%= setting(:abbreviation) %> papers are subject to the <a
             href="https://opensource.org/licenses/MIT" target="_blank">MIT license</a> regardless of the license of the
           submitted software package under review.</p>
-        <p>Any use of the JOSS logo is licensed CC BY 4.0.</p>
+        <p>Any use of the JOSS logo is licensed CC BY 4.0. See the logo directory in the <a rel="https://github.com/openjournals/digital-assets">digital-assets</a>-repository for more information about it.</p>
```

Comment:
Can you change this to "See the joss/logo directory ..."

Reply:
Done.